src/mapped-types.ts

```diff
@@ -36,6 +52,10 @@ export type NonUndefined<A> = A extends undefined ? never : A;
 /**
  * FunctionKeys
  * @desc get union type of keys that are functions in object type `T`
+ * @example
+ *   // Expect: "setName"
```

Comment:
Expect comment line should be above the tested type, it's FunctionKeysProps in this case.

Before it was line below the tested type, but I think it would be better to have it line above the tested type as it is a convention used in many type testing frameworks.

Please follow up with this convention in all the examples below

Reply:
done, PTAL